### PR TITLE
[storybook] 404

### PR DESCRIPTION
**Pull request by dvdherron**
## Cute animal pic
image

_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

## Link to Trello card (if applicable)
https://trello.com/c/f0jSvig0/93-metecho-404
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

## Description
_The commit messages say what you did; this should explain why and/or how._
Adds general explanatory documentation to the 404 component. 

- [x ] Description is in Trello card

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._
Documention currently lies in the "Intro" section under the 404 component

## Show me
_Provide screenshots/animated gifs/videos if necessary._
image

### CODE REVIEW

**Comment by dvdherron** [comment]
[Metecho › 404](https://trello.com/c/f0jSvig0/93-metecho-404)

**Review by stacyk — APPROVED** [review]
LGTM!

**Review by duggiemitchell — APPROVED** [review]
I added a little more info on the docs page. LGTM!

File: src/stories/pages/404.stories.tsx [hunk 1]
```diff
@@ -12,9 +12,15 @@ const Template: Story<ComponentProps<typeof FourOhFourComponent>> = (args) => (
   <FourOhFourComponent {...args} />
 );
 
+const Message404 = () => (
+  <>
+    We can’t find the epic you’re looking for. Try another epic from{' '}
+    <a href="#">this projects</a>
```
Comment (jgerigmeyer) [inline src/stories/pages/404.stories.tsx:18]:
```suggestion
    <a href="#">this project</a>.
```

**Pull request merged**